### PR TITLE
Ssant096/landing animation

### LINKED ISSUE #123: Landing Component Animation
image

- Use framer motion to animate the transitions for the website
- Image should not be affected by animation.
- In the animation the title and location should fade in at the same time, later the button should fade in. (0.2 seconds delay).
- Example, see the landing component of the PAD website: https://pad.ucrhighlanders.org/

### PR DESCRIPTION
**Pull request by ssant096**
ISA Landing Animation

### CODE REVIEW

File: src/components/home/Landing.jsx [hunk 1]
```diff
@@ -1,21 +1,55 @@
...
+        <motion.div
+          variants={animations}
+          initial="start"
+          whileInView="end"
+          transition={{ delay: 0.2 }}
+        >
+          Indian Student Association
+        </motion.div>
         <div className="text-isa-white md:text-3xl text-sm flex flex-row">
           <FaMapPin className="mr-5" />
```
Comment (seanquiambao) [inline src/components/home/Landing.jsx]: Include the pin in the motion divider

**Review by seanquiambao — CHANGES_REQUESTED** [review]
Unstage the board/page.js, layout.js and not-found.js in this branch. I think you might have brought over your previous issues into this branch.

File: src/components/home/Landing.jsx [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
         <div className="text-isa-white md:text-3xl text-sm flex flex-row">
-          <FaMapPin className="mr-5" />
-          University of California Riverside
+          <motion.div
+            variants={animations}
+            initial="start"
+            whileInView="end"
+            transition={{ delay: 0.2 }}
+          >
+            <FaMapPin className="mr-5" />
```
Comment (seanquiambao) [inline src/components/home/Landing.jsx:41]: you can put the FaMap Pin in the same motion div in the location

**Review by seanquiambao — CHANGES_REQUESTED** [review]

File: src/components/home/Landing.jsx (hunk 2 above)
Comment (ssant096) [reply src/components/home/Landing.jsx:41]: When I do that the pin and text misalign for some reason:
image

Comment (seanquiambao) [reply src/components/home/Landing.jsx:41]: you can edit the className in motion.div and use Tailwind CSS to align the pin correctly.

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**